store/views.py

```diff
@@ -73,10 +96,51 @@ def loanBookView(request):
 You need to accept the book id as argument from a post request.
 You additionally need to complete the returnBook function in the loaned_books.html file
 to make this feature complete
-''' 
+'''
 @csrf_exempt
 @login_required
+@require_http_methods(["POST"])
 def returnBookView(request):
-    pass
+    response_data = {
+        'message': None,
+    }
+    book_id=request.POST['bid']
+    try:
+        book=BookCopy.objects.get(pk=book_id)
+        book.status=True
+        book.borrower=None
+        book.borrow_date=None
+        book.save()
+        response_data['message']='Successfully Book returned.'
+        return JsonResponse(response_data)
 
+    except:
+        response_data['message']='No such book_id found'
+        return JsonResponse(response_data)
 
+@csrf_exempt
+@login_required
+@require_http_methods(["POST"])
+def ratingItView(request,bid):
+    response_data = {
+        'message': None,
+    }
+    book=Book.objects.get(pk=bid)
+    ratinginp=float(request.POST.get('rat'))
+    ratinginp=int(ratinginp)
+    ratinginp=max(0,ratinginp)
+    ratinginp=min(10,ratinginp)
```

Comment:
You should not adjust the value of rating to lie between 0 and 10. Rather you should alert the user in case he rates the book beyond the allowed range.